Source/Filter Effects/feGaussianBlur/SvgGaussianBlur.cs

```diff
@@ -222,23 +231,12 @@ public Bitmap Apply(Image inputImage)
         /// Gets or sets the radius of the blur (only allows for one value - not the two specified in the SVG Spec)
         /// </summary>
         [SvgAttribute("stdDeviation")]
-        public float StdDeviation
+        public SvgNumberCollection StdDeviation
         {
-            get { return _stdDeviation; }
-            set
-            {
-                if (value <= 0f)
-                {
-                    throw new InvalidOperationException("Radius must be greater then 0");
-                }
-                _stdDeviation = value;
-                Attributes["stdDeviation"] = value;
-                PreCalculate();
-            }
+            get { return GetAttribute("stdDeviation", false, new SvgNumberCollection() { 0f }); }
+            set { Attributes["stdDeviation"] = value; }

```

Comment:
Negative number check has been removed.
What happens if negative number is set ?

FYI:
Firefox: Do not display element.
Chrome: No effect of filter effect.

Reply:
It shouldn't throw in setter. It should be handled in filter processing part.

Reply:
What kind of image is output when negative number is set?

Reply:
It's error but I don't think throwing in setter is the right place to handle such errors. 

There are various ways proposed here:
https://www.w3.org/TR/SVG11/implnote.html#ErrorProcessing

Reply:
There is no problem with deletion.

Reply:
Behavior is as follows.

### Test SVG

```svg
<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" width="120" height="120">
  <defs>
    <filter id="f1" x="0" y="0">
      <feGaussianBlur in="SourceGraphic" stdDeviation="-1" />
    </filter>
  </defs>
  <rect x="10" y="10" width="90" height="90" stroke="green" stroke-width="2" fill="yellow" filter="url(#f1)" />
</svg>
```

### Before PR

![before](https://user-images.githubusercontent.com/11144112/73580149-e5096400-44c7-11ea-8974-5ee826167a1f.png)

### After PR

![output](https://user-images.githubusercontent.com/11144112/73580165-ecc90880-44c7-11ea-8a06-0c57ae1dd9b5.png)

become like firefox.

Reply:
https://github.com/vvvv/SVG/pull/659